packages/casting/src/casting-spec.js

```diff
@@ -28,19 +29,40 @@ const vstoragePathToCastingSpec = (storagePath, storeName = 'vstorage') => {
   const buf = toAscii(`${elems.length}.${storagePath}`);
   return harden({
     storeName,
-    storeSubkey: buf.map(b => (b === PATH_SEPARATOR_BYTE ? 0 : b)),
+    storeSubkey: buf.map(b =>
+      b === PATH_SEPARATOR_BYTE ? KEY_SEPARATOR_BYTE : b,
+    ),
     dataPrefixBytes: DATA_PREFIX_BYTES,
   });
 };
 
+// TODO make a similar castingSpecToPath.
+// This one uses VStorageKey which has storeSubkey:string, not UInt8Array
```

Comment:
@michaelfig it turned out that the `storeSubkey` here was "string", not UInt8Array as in a legit CastingSpec. So I wrote `vstorageKeySpecToPath` and left this note with your idea to have `castingSpecToPath`.